src/Commands/AzureBestPractices/AzureBestPracticesGetCommand.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using AzureMcp.Arguments;
+using AzureMcp.Models.Command;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+
+
+namespace AzureMcp.Commands.BestPractices;
+
+public sealed class AzureBestPracticesGetCommand(ILogger<AzureBestPracticesGetCommand> logger) : BaseCommand()
+{
+    private readonly ILogger<AzureBestPracticesGetCommand> _logger = logger;
+
+    private static readonly string _bestPracticesText = LoadBestPracticesText();
+
+    private static string GetBestPracticesText() => _bestPracticesText;
+
+    private static string LoadBestPracticesText()
+    {
+        var assembly = typeof(AzureBestPracticesGetCommand).Assembly;
+        const string resourceName = "AzureMcp.Resources.azure-best-practices.txt";
+
+        using var stream = assembly.GetManifestResourceStream(resourceName)
+            ?? throw new InvalidOperationException($"Unable to load embedded resource: {resourceName}");
+
+        using var reader = new StreamReader(stream);
+        return reader.ReadToEnd();
+    }
+
+    protected override string GetCommandName() => "get";
+
+    protected override string GetCommandDescription() =>
+        "[LLM-MANDATORY] Returns best practices for secure, production-grade Azure SDK usage. " +
+        "Call this command and follow its guidance before generating or suggesting any Azure SDK code. " +
+        "This command returns the content of the markdown file as a string array. " +
+        "#llm #best-practices #code-generation #azure #mandatory";
```

Comment:
Do these hash tags get parsed by the client?  Are they helpful?  Do we have evidence of that?

Reply:
These were something I was experimenting with, doesn't seem to create any additional impact, words description works fine.

Reply:
removed the tags.